docs/root/operations/traffic_capture.rst

```diff
@@ -0,0 +1,70 @@
+.. _operations_traffic_capture:
+
+Traffic capture
```

Comment:
I can add a comment, but TBH, Envoy itself is a data exfiltration feature if misconfigured, and the only way this gets turned on is if you configure it ;)

Reply:
Good point :) The recently announced twitter issue where cleartext passwords were accidentally stored (presumably through logging feature of some sort, because I really don't think they were intentionally storing cleartext passwords) was fresh on my mind when I was thinking about this.